app/routes/events/components/EventAdministrate/Allergies.tsx

```diff
@@ -0,0 +1,155 @@
+import moment from 'moment-timezone';
+import { Link } from 'react-router-dom';
+import { Flex } from 'app/components/Layout';
+import LoadingIndicator from 'app/components/LoadingIndicator';
+import Table from 'app/components/Table';
+import type {
+  Event,
+  EventPool,
+  ActionGrant,
+  User,
+  ID,
+  EventRegistration,
+  EventRegistrationPaymentStatus,
+  EventRegistrationPresence,
+} from 'app/models';
+import type Comment from 'app/store/models/Comment';
+import type { CurrentUser } from 'app/store/models/User';
+import { PresenceIcons } from './AttendeeElements';
+import { RegistrationPill, getRegistrationInfo } from './RegistrationTables';
+
+export type Props = {
+  eventId: number;
+  event: Event;
+  comments: Comment[];
+  pools: Array<EventPool>;
+  loggedIn: boolean;
+  currentUser: CurrentUser;
+  error: Record<string, any>;
+  loading: boolean;
+  registered: Array<EventRegistration>;
+  unregistered: Array<EventRegistration>;
+  unregister: (arg0: {
```

Comment:
Is it possible to get a better name than arg0?

Reply:
I just now pushed a new change where I removed both the unregister and updatePresence (comment below) all together as they are not used in the allergies tab.

Reply:
I have now fixed these issues in the original Attendes file as well as some deprecated syntax